### PR TITLE
Add revert screen position worker

### PR DESCRIPTION
**Pull request by ivan-ha**

### CODE REVIEW

File: worker/random_score.js [hunk 1]
```diff
@@ -6,9 +6,16 @@ const Promise = require('bluebird');
...
+if (process.argv.length < 4) {
+    console.log('Usage: node random_score.js [brand] [cinemas]');
+}
 
-const cinema_dirs = ['tsing-yi','citywalk'];
-const data_path = path.resolve(__dirname, '../', 'backend/data','gh');
+const BRAND = process.argv[2];
+var cinema_dirs = [];
+for (var i = 3; i < process.argv.length; i++) {
+    cinema_dirs.push(process.argv[i]);
```
Comment (samMeow) [inline worker/random_score.js]: why not simply
```js
const cinema_dirs = process.argv.slice(3)
```

File: worker/revert_screen.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+if (process.argv.length < 4) {
+    console.log('Usage: node revert_screen.js [brand] [cinemas]');
+}
+
+const DATA_PATH = '../backend/data';
+const BRAND_NAME = process.argv[2];
+var cinemas = [];
+for (var i = 3; i < process.argv.length; i++) {
+    cinemas.push(process.argv[i]);
```
Comment (samMeow) [inline worker/revert_screen.js]: same

File: worker/random_score.js [hunk 3]
```diff
@@ -6,9 +6,16 @@ const Promise = require('bluebird');
 
 Promise.promisifyAll(fs);
 
+if (process.argv.length < 4) {
+    console.log('Usage: node random_score.js [brand] [cinemas]');
```
Comment (samMeow) [inline worker/random_score.js:10]: should have exit statement
```js
process.exit()
```

**Review by samMeow — CHANGES_REQUESTED** [review]

File: worker/random_score.js (hunk 1 above)
Comment (ivan-ha) [reply worker/random_score.js]: fucking right😂
will change it tonight

**Comment by ivan-ha** [comment]
done @samMeow

**Review by samMeow — APPROVED** [review]

**Pull request merged**